test/vm.install

```diff
@@ -23,6 +23,13 @@ Delegate=cpu cpuset io memory pids
 EOF
 fi
 
+if grep -q platform:el10 /usr/lib/os-release && [ -e /etc/firewalld/firewalld.conf ]; then
```

Comment:
Do we need to restrict this to rhel-10? We could just erase StrictForwardPort settings everywhere.

Reply:
Also, could we install a config snippet that overrides StrictPortForward, instead of having to know which file has the changed OS default?

Reply:
> Do we need to restrict this to rhel-10

We don't strictly have to, but as the default is explicitly "no" on Fedora and everywhere else, and this is currently a RHEL 10-ism, I'd rather restrict the hack to that, and learn if it creeps in anywhere else. TBH this is so ridiculously broken that I don't expect this to stay around for long -- this needs to go back to the drawing board.

> could we install a config snippet

I'd love to, and that's usually the first thing I look for. Changing big distro-provided conffiles is baaad. But firewalld.conf doesn't seem to have anything that resembles an "include conf.d", nor is there any directory like that. But again, I expect this to be a temporary hack.